### PR TITLE
Tracking changes for superlike-integration

### PR DESCRIPTION
**Pull request by mayeaux**
Adding superlike integration

### CODE REVIEW

File: desktop-app/package.json [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+  "single-instance": false,
   "user-agent": "Mozilla/5.0 (Linux; U; en-gb; KFTHWI Build/JDQ39) AppleWebKit/535.19 (KHTML, like Gecko) Silk/3.16 Safari/535.19",
   "no-edit-menu": true,
   "dependencies": {
     "moment": "^2.8.4",
-    "tinderjs": "git://github.com/mfkp/tinderjs",
-    "nw-updater": "git://github.com/mfkp/nw-updater"
+    "nw-updater": "git://github.com/mfkp/nw-updater",
+    "tinder": "^1.10.0",
+    "tinderjs": "git://github.com/mfkp/tinderjs"
```
Comment (SaturnusDJ) [inline desktop-app/package.json:25]: Should be `"tinderjs": "git://github.com/tinderjs/tinderjs"` now?

**Comment by mayeaux** [comment]
Nope, that `tinder` module is now the `tinderjs` module. I got the npm module for `tinder` if you can believe it bahaha. I'll remove the `tinderjs` module completely once I'm completely sure we're not using it anywhere.

**Comment by SaturnusDJ** [comment]
Aha! So https://github.com/tinderjs/tinderjs will be renamed to tinder?

**Comment by mayeaux** [comment]
Maybe, maybe not. Haven't quite decided yet. Will probably leave it as `tinderjs` actually as its clearer, I just needed a new npm module so people could install it that way, and was able to get my hands on the `tinder` module name :)

**Comment by mayeaux** [comment]
Okay superlike is working! Couple extra bells and whistles to throw in there before I'm happy but it _is_ functional

**Comment by mayeaux** [comment]
Testing something

**Pull request merged**